src/integrationTest/java/com/mongodb/hibernate/type/temporal/SqlTimeIntegrationTests.java

```diff
@@ -62,9 +62,6 @@ void unsupported() {
                 () -> assertNotSupported(
                         new ItemWithNestedAggregateEmbeddableWithArrayPersistentAttribute<Time>() {}.getClass()),
                 () -> assertNotSupported(
-                        new ItemWithNestedAggregateEmbeddableWithCollectionPersistentAttribute<Time>() {}.getClass()),
```

Comment:
Why was this test case removed?

The same question applies to `SqlTimestampIntegrationTests`, `ZonedDateTimeIntegrationTests`.